test/unit/UFragmentsPolicy.js

```diff
@@ -292,6 +293,14 @@ contract('UFragmentsPolicy:Rebase:accessControl', async function (accounts) {
       ).to.be.true;
     });
   });
+
+  describe('when rebase called by a contract which is being constructed', function () {
+    it('should fail', async function () {
+      expect(
+        await chain.isEthException(ConstructorRebaseCallerContract.new(uFragmentsPolicy.address))
```

Comment:
We should use the same pattern as above, but this one's shorter. Might as well update the other test function to not store `rebaseCallerContract` etc.

Reply:
Done 👍